package.json

```diff
@@ -61,7 +61,8 @@
     "nyc": "^15.0.0",
     "publish-please": "^5.4.3",
     "sinon": "^1.17.4",
-    "sinon-chai": "^2.8.0"
+    "sinon-chai": "^2.8.0",
+    "babel-cli":  "^6.26.0"
   },
   "engines": {
     "node": ">=0.12",
```

Comment:
I think we should be using node v14 as our engine, but declare somewhere else that it work work with v14 and lower up to v0.12 with no guarantee, the reason being is this a public facing code and NodeJS can't help when there are vulnerability issues in the future cause it's out of support please refer to here https://nodejs.org/en/about/releases/ @skoong and @gmeyer-hw something to consider in Risk management perspective

Reply:
@peter-joseph agreed that we should be more specific about which version of node that the sdk will work on.  Is this something that we can change on the fly like this or do we need some more approvals?

Reply:
Seems like our doc is saying 6.15.1 right now - https://docs.hyperwallet.com/content/api/v3/overview/sdk#node

Reply:
it depends on the target release version, https://nodejs.dev/learn/semantic-versioning-using-npm, cause if this is a breaking change then integrators with setup `^2.x.x` will be updated if integrators, issue `npm update` and the release is minor or patch, and if we change other than the documentation then we need to loop in product